CodePush.js

```diff
@@ -302,7 +307,7 @@ async function syncInternal(options = {}, syncStatusChangeCallback, downloadProg
       resolvedInstallMode = localPackage.isMandatory ? syncOptions.mandatoryInstallMode : syncOptions.installMode;
       
       syncStatusChangeCallback(CodePush.SyncStatus.INSTALLING_UPDATE);
-      await localPackage.install(resolvedInstallMode, () => {
+      await localPackage.install(resolvedInstallMode, syncOptions.minimumBackgroundDuration, () => {
```

Comment:
Should we do

`typeof syncOptions.minimumBackgroundDuration === "number" ? syncOptions.minimumBackgroundDuration : 0`

to prevent an exception?